specs/credentialmanagement/index.src.html

```diff
@@ -495,16 +512,17 @@ <h4 id="interfaces-credential-types-credential"><code>Credential</code></h4>
     interface Credential {
       readonly attribute DOMString id;
       readonly attribute DOMString type;
+      boolean isMatch (any filter);
```

Comment:
Nit: I'd prefer `matches`, but I guess it's still not clear to me what this is intended to model for credentials that aren't stored locally. That is, I understand the use case for sifting through locally stored passwords and federation data. I don't see how this "filtering" model fits with the kinds of cross-origin, third-party data that it seems it's meant to enable.